cli/pkg/backend/aws/backend.go

```diff
@@ -215,6 +215,10 @@ func NewAWSBackend(
 	return b, nil
 }
 
+func (b Backend) GetCredentials() (aws.Credentials, error) {
+	return b.awsConfig.Credentials.Retrieve(context.Background())
```

Comment:
we should probably pass a context in here and then use the CLI's context that comes from that CLI library